pkg/publish_links.go

```diff
@@ -36,7 +36,8 @@ func getAnchorHrefsFromHTML(content string) ([]string, error) {
 			}
 		}
 	}
-	return anchorHrefs, nil
+	// Commented out because go vet complains that this in unreachable.
+	// return anchorHrefs, nil
```

Comment:
How do we actually want this code to be structured?  Generally for good code readability it's recommended to restructure loops so they can have their breaking condition in the header/footer condition rather than a break/return statement in the middle.

Reply:
Yup. I've got plans which are out-of-scope for this changeset.